console/src/components/AppState.js

```diff
@@ -8,11 +8,10 @@ const saveAppState = state => {
   window.localStorage.setItem(stateKey, JSON.stringify(state));
 };
 
-const loadAccessKeyState = () =>
-  window.localStorage.getItem('CONSOLE_ACCESS_KEY');
+const loadAccessKeyState = () => window.localStorage.getItem(stateKey);
```

Comment:
Changed it to `stateKey` it was `CONSOLE_ACCESS_KEY` any reasons? same for saveAccessKeyState also.

Reply:
Revert back to 'CONSOLE_ACCESS_KEY' because stateKey variable is used for other localstorage info like history.